paper/paper.md

```diff
@@ -0,0 +1,41 @@
+---
+title: 'JupyterLab_Voyager: A Data Visualization Enhancement for Jupyterlab'  
+tags:
+  - JupyterLab
+  - Voyager
+  - Vega-Lite
+  - Altair
+  - data visualization   
+
+authors:
+  - name: Ji Zhang  
+    orcid: 0000-0003-1599-5437  
+    affiliation: 1  
+  - name: Brian Granger  
+    orcid:   
+    affiliation: 1  
+  - name:   
+    orcid:   
+    affiliation: 1  
+    
+affiliations:
+ - name: College of Engineering, California Polytechnic State University  
+   index: 1  
+   
+date: XX June 2018  
+bibliography: paper.bib
+---
+
+# Summary
+
+Scientific data analysis and visualization (DAV) tools are critical components of the data science software ecosystem; the usability of these tools is becoming extremely important to facilitate next-generation scientific discoveries. JupyterLab has been considered as one of the best polyglot, web-based, open-source data science tools [@perez:2015]. As the next phase of extensible interface for the classic iPython notebooks, this tool supports interactive data science and scientific computing across multiple programming languages with great performances[@kluyver:2016]. Despite these advantages, previous heuristics evaluation studies [@swaid:2017] have shown that JupyterLab has some significant flaws in the data visualization side. The current DAV system in JupyterLab heavily relies on users understanding and familiarity with certain visualization libraries, and doesnt support the golden visual-information-seeking mantra of overview first, zoom and filter, then details-on-demand [@shneiderman:1996]. These limitations often lead to a workflow bottleneck at the start of a project.
```

Comment:
JupyterLab has significant flaws in constructing data visualizations.

Reply:
The current DAV system in JupyterLab heavily relies on the user's understanding and familiarity with certain visualization libraries and does not support the golden visual-information-seeking mantra of overview first, zoom and filter, then details-on-demand.